wbooks/src/components/book-detail/book-comments/BookComments.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+import styles from './BookComments.styles.scss';
+import AddComment from './add-comment/AddComment';
+import CommentList from './comment-list/CommentList';
+
+class BookComments extends Component {
+  render() {
+    return (
+      <div>
```

Comment:
empty divs are working? I think that styles are required

Reply:
![screen shot 2018-02-07 at 14 25 12](https://user-images.githubusercontent.com/7648908/35931201-ba0ecbcc-0c12-11e8-8981-a849d66138b5.png)

Apparently its working but what would you add?

Reply:
in gral, divs container has flex: 1, this is working has if have flex: 1  jaj.
Check how work on responsive